src/utils/openmc/executor.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+from openmc.checkvalue import *
+import subprocess
+import os
+
+
+FNULL = open(os.devnull, 'w')
```

Comment:
It is really bad to keep a file handle open - even devnull - for the whole Python process like this.